views/cypress/utils/set-response.js

```diff
@@ -116,3 +118,111 @@ export function setResponse(widgetSelector, widgetActiveSelector, qtiClass) {
 
     cy.get(`${widgetActiveSelector} button.widget-ok`).click({ force: true });
 }
+
+function previewItem () {
+    cy.intercept('GET', urls.itemPreview).as('preview');
+    cy.get(selectors.previewItemButton).should('not.have.class', 'disabled');
+    cy.get(selectors.previewItemButton).click({force :true});
+    cy.wait('@preview');
+    cy.get('.qti-choiceInteraction').should('exist');
+}
+
+function saveItem() {
+    cy.intercept('POST', '**/saveItem*').as('saveItem');
+    cy.get('[data-testid="save-the-item"]').click();
+    cy.wait('@saveItem')
+        .its('response.body')
+        .its('success')
+        .should('eq', true);
+}
+
+export function addResponseProcessing(
+    widgetSelector,
+    qtiClass,
+    responseProcessingOptions
```

Comment:
Should not be better to name this param `responseProcessingOption`? It contains an option, not an array of options. Missing JSDoc will help here.

Reply:
change added here 9714d687946b1af142470d6271821d2478836b64[](url)